duckduckgo/Android
# Bookmarks

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/0/353085934262754
CC:

**Description**: Implement basic bookmarks in the app


**Steps to test this PR**:
Test 1
1. Load web page
2. Select Save Bookmark from the menu
3. Open Bookmarks and see the just saved bookmark

Test 2
1. Save a few bookmarks
2. Try to open each bookmark and confirm the browser loads the right url for each bookmark

Test 3
1. Press Edit on the Bookmark menu
2. Delete some bookmark by pressing the delete button or swiping it.
3. Confirm that the bookmark are properly deleted

Test 4
1. Press Edit on the Bookmark menu
2. Select a single Bookmark and edit the proper info
3. Confirm that the bookmark is edited, saved properly and open the right url

Test 5
1. Force close the app
2. Confirm the bookmarks are preserved when reopening the app


###### Reviewer Checklist:


###### PR DRI Checklist:
